main_test.go

```diff
@@ -132,6 +132,7 @@ func TestParseAndRunConfig(t *testing.T) {
 				"PLUGIN_MEMORY":                "128Mi",
 				"PLUGIN_TIMEOUT":               "10s",
 				"PLUGIN_REGION":                "us-central1",
+				"PLUGIN_CLOUD_SQL_UPDATE":      "-my-proj:east2:db2",
```

Comment:
Can you also add a tests the error case? E.g. with an invalid CloudSQL string with `first char != {+-=c}` ?

Reply:
:+1: